woocommerce/woocommerce-ios
# Fix/speculative fix for double order pushes

Part of: #12071


## Description


Some of the issues under this crash appear to be related to double-pushing Order loaders, probably from a notification.

This _really_ shouldn't be possible, because we make a new instance of the order loader every time that we use it, and we also don't push them, we set the splitview's viewcontroller's property to an array with just the one order loader in it.

So... I'm not confident that this fix will do anything, but it doesn't break anything either.

If this doesn't have any effect on the crash numbers, we should remove it

peaMlT-zn-p2 for context.

## Testing instructions


I could not repro the crash, so these are general testing instructions.

1. Launch the app, then background it
2. Make an order on the web
3. Tap the notification which comes up
4. Observe that the order is shown as expected.


---
